### ISSUE TITLE
Some systems failed to build for Quicklisp dist

### ISSUE THREAD

**Issue by quicklisp** [y2q-actionman/with-c-syntax#14]
Building with SBCL 2.0.5 / ASDF 3.3.1 for quicklisp dist creation.

Trying to build commit id b0429e8cb4565365f90d3e30b7d964d02e115c95

*with-c-syntax-test* fails to build with the following error:

```
Unhandled UNDEFINED-FUNCTION in thread #<SB-THREAD:THREAD "main thread" RUNNING {1000A18083}>: The function WITH-C-SYNTAX.LIBC:|ilogb| is undefined.

```

*with-c-syntax* fails to build with the following error:

```
Unhandled UNDEFINED-FUNCTION in thread #<SB-THREAD:THREAD "main thread" RUNNING {1000A18083}>: The function WITH-C-SYNTAX.LIBC:|ilogb| is undefined.

```

[Full log here](http://report.quicklisp.org/2021-06-01/failure-report/with-c-syntax.html)

**Comment by y2q-actionman** [comment]
Confirmed. I will fix it soon.

**Comment by y2q-actionman** [comment]
It should be fixed by commit c73e5259
https://github.com/y2q-actionman/with-c-syntax/commit/c73e5259d0559681545d6085119e28365a4744aa#diff-f6f8faa6108671264fe3f7954cdbf22a4e35e74ea7e11bc42bc1c921670ed98bR161-R162
